src/ApplyOfflineMap.cpp

```diff
@@ -307,8 +372,15 @@ try {
 			vecOutputDataFiles[f],
 			vecVariableStrings,
 			strNColName,
+			OverlapMesh,
```

Comment:
`lb, ub` should be computed before the call to `mapRemap.Apply` here for each of vecInputDataFiles; refer to previous comment